livox_ros2_driver/livox_ros2_driver/lddc.cpp

```diff
@@ -694,6 +694,7 @@ void Lddc::initializeDiagnostics()
   updater_.add("livox_fan_status", this, &Lddc::checkFan);
   updater_.add("livox_ptp_signal", this, &Lddc::checkPTPSignal);
   updater_.add("livox_time_sync", this, &Lddc::checkTimeSync);
+  updater_.add("livox_connect", this, &Lddc::checkConnect);
```

Comment:
Could you use `livox_connection` instead?

Reply:
(and `checkConnection`)

Reply:
done